device-types/Extreme Networks/5420F-48T-4XE.yaml

```diff
@@ -0,0 +1,192 @@
+    mgmt_only: false
+  - name: '9'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '10'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '11'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '12'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '13'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '14'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '15'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '16'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '17'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '18'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '19'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '20'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '21'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '22'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '23'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '24'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '25'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '26'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '27'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '28'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '29'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '30'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '31'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '32'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '33'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '34'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '35'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '36'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '37'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '38'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '39'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '40'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '41'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '42'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '43'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '44'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '45'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '46'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '47'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '48'
+    type: 1000base-t
+    mgmt_only: false
+  - name: '49'
+    type: 10gbase-x-sfpp
+    mgmt_only: false
+  - name: '50'
+    type: 10gbase-x-sfpp
+    mgmt_only: false
+  - name: '51'
+    type: 10gbase-x-sfpp
+    mgmt_only: false
+  - name: '52'
+    type: 10gbase-x-sfpp
+    mgmt_only: false
+  - name: '53'
+    type: other
+    mgmt_only: false
+    description: either Stacking-Ports or additional SFP+
+  - name: '54'
+    type: other
+    mgmt_only: false
+    description: either Stacking-Ports or additional SFP+
+  - name: Mgmt
+    type: 1000base-t
+    mgmt_only: true
+device-bays:
+  - name: PSU-2
```

Comment:
PSU-2 is a plugable power supply. You can see a picture in [figure 2](https://documentation.extremenetworks.com/HW_QRG/5420_QuickReference.pdf). 

I'm not sure which is the best approach. I simply copied from one existing device:

- 5520-12MW-36W-FabricEngine.yaml uses module bays for this kind of device
- 5520-24T.yaml uses device bays 
- X460-G2-24t-10GE4.yaml also uses device bays

Let me know what you prefer

Reply:
I found your answer in another comment. I'll change them to `module-bay`

Reply:
Can `device-bays` have `airflow`? Because x450 and x460 have plugable fan modules with either front-to-back or back-to-front. Should I create 2 `module-types` for these fans?

I don't know if I'm being to chatty, making too many comments ot watever. This is my first PR to this project. So please guide me.

Reply:
Power supplies are not devices, but rather modules. PSUs should be under `module-bays`

Reply:
They already are, on 5420. Changed it on x450 and x460